pom.xml

```diff
@@ -140,37 +140,152 @@
+        <version.org.jboss.shrinkwrap>1.0.1</version.org.jboss.shrinkwrap>
+        <version.org.jboss.spec.javax.ejb>1.0.0.Final</version.org.jboss.spec.javax.ejb>
+        <version.org.jboss.spec.javax.jms>1.0.0.Final</version.org.jboss.spec.javax.jms>
+        <version.org.jboss.spec.javax.transaction>1.0.1.Final</version.org.jboss.spec.javax.transaction>
+        <version.org.jboss.stdio>1.0.1.GA</version.org.jboss.stdio>
+        <version.org.jboss.threads>2.3.2.Final</version.org.jboss.threads>
+        
+        <version.org.slf4j>1.7.2</version.org.slf4j>
+        <version.org.springframework.boot>2.0.9.RELEASE</version.org.springframework.boot>
+        <version.org.springframwork>5.2.22.RELEASE</version.org.springframwork>
+        <version.org.wildfly.arquillian>4.0.0.Alpha5</version.org.wildfly.arquillian>
+        <version.plugin.maven.dependency>2.3</version.plugin.maven.dependency>
+        <version.postgresql>9.1-901.jdbc4</version.postgresql>
+        <version.resteasy.api>3.0.5.Final</version.resteasy.api>
+        <version.resteasy.client>4.6.0.Final</version.resteasy.client>
+        <version.resteasy.jaxrs>4.0.0.Beta5</version.resteasy.jaxrs>
+        <version.resteasy>3.0.23.Final</version.resteasy>
+        <version.shrinkwrap>1.2.6</version.shrinkwrap>
+        <version.surefire.plugin>2.19.1</version.surefire.plugin>
+        <version.tomcat.plugin>2.2</version.tomcat.plugin>
+        <version.transaction.api>1.0.1.Final</version.transaction.api>
+        <version.war.plugin>3.2.3</version.war.plugin>
+        <version.weld>3.1.9.Final</version.weld>
+        
+        <version.wildfly.plugin>1.1.0.Alpha10</version.wildfly.plugin>
+        <version.wildfly.core>18.1.2.Final</version.wildfly.core>
+        <version.wildfly.logging>5.0.0.Beta5</version.wildfly.logging>
+        
   </properties>
 
-  <profiles>
-    <profile>
-      <id>jdk9</id>
-      <activation>
-        <jdk>[9,)</jdk>
-      </activation>
-      <properties>
-        <modular.jdk.args>--add-exports=java.base/sun.nio.ch=ALL-UNNAMED
-          --add-exports=jdk.unsupported/sun.reflect=ALL-UNNAMED
-          --add-exports=jdk.unsupported/sun.misc=ALL-UNNAMED
-          --add-modules=java.se</modular.jdk.args>
-      </properties>
-    </profile>
-    <profile>
-      <id>include-for-JDK11-above-versions</id>
-      <activation>
-        <jdk>[11,)</jdk>
-      </activation>
-      <modules>
-        <module>jta-1_2-in-wildfly</module>
-        <module>wildfly/commit-markable-resource</module>
-      </modules>
-    </profile>
-  </profiles>
+  <build>
+    <pluginManagement>
+      <plugins>
+            <plugin>
+              <artifactId>maven-war-plugin</artifactId>
+              <version>${version.war.plugin}</version>
+              <configuration>
+                <archive>
+                  <index>true</index>
+                </archive>
+                <failOnMissingWebXml>false</failOnMissingWebXml>
```

Comment:
Wasn't this there only because to JDK6?

Reply:
Thanks @jmfinelli for the question. Actually it is needed also with jdk17 for some quickstarts packaged as war. Do you think we should use this configuration only with specific projects that need it?